tests/serialise/openconfig-serialise/json/interfaces_ph.False-flt.False-m.default.json

```diff
@@ -13,7 +13,7 @@
                     }
                 }, 
                 "config": {
-                    "enabled": true, 
+                    "enabled": 1, 
```

Comment:
How come we now have different boolean representations, I would expect these to be:
"enabled": true